### DISCUSSION TITLE
“Evaluate in debug console” command is missing

### DISCUSSION THREAD

**Discussion by chnlior** [eclipse-theia/theia#10279] [General] [answered]
Hi,
I am using v1.17.2(beta) on macos

Vscode has a command **“Evaluate in debug console”** (see [here](https://github.com/microsoft/vscode/issues/98390)).
It seems like Theia does not have this command, so I basically need to copy and paste the code to the debug console and then evaluate it (I am using python).

Am I missing something? Is there a way to enable this capability?

**Comment by vince-fugnitto** [comment, marked as answer]
@chnlior did you mean to open an issue? (duplicate of https://community.theia-ide.org/t/vscode-evaluate-in-debug-console-command-is-missing/1997)

**Reply by chnlior** [reply]
Yes, but first wanted to be positive i am missing something and it is indeed an issue.

**Reply by vince-fugnitto** [reply]
@chnlior yes I confirmed yesterday that the command is not yet supported (`editor.debug.action.selectionToRepl`) which is why I mentioned opening the issue :+1: 
- [vscode implementation](https://github.com/microsoft/vscode/blob/c132e4e19bd3e3515c944585a3257816be8d3928/src/vs/workbench/contrib/debug/browser/debugEditorActions.ts#L217-L245)

**Reply by chnlior** [reply]
oh, sorry for misunderstood you. 
I will do it now.
